Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -368,9 +355,9 @@ extension XPCMachServer {
             return false
         }
         
-        // Login item must be within the main app bundle's Contents/Library/LoginItems directory
+        // Login item must be within the main bundle's Contents/Library/LoginItems directory
         let pathComponents = loginItem.deletingLastPathComponent().pathComponents
-        guard pathComponents.count >= 3, pathComponents.suffix(3) == ["LoginItems", "Library", "Contents"] else {
+        guard pathComponents.count >= 3, pathComponents.suffix(3) == ["Contents", "Library", "LoginItems"] else {
```

Comment:
heh, nice catch.

Reply:
Yup 😬

Wish there was a good way to test these in automated way, but short of a sophisticated integration test I don't think it's going to happen. And setting up and maintaining that seems prohibitive.